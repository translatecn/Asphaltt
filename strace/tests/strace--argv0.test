#!/bin/sh -efu
#
# Check decoding of --argv0 option.
#
# Copyright (c) 2023 The strace developers.
# All rights reserved.
#
# SPDX-License-Identifier: GPL-2.0-or-later

. "${srcdir=.}/init.sh"

args=
run_prog ../fchdir > /dev/null
run_strace --argv0=sample --trace=execve $args > /dev/null
match_grep
