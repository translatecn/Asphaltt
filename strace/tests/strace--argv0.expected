execve\("\.\./fchdir", \["sample"\], 0x[[:xdigit:]]* /\* [[:digit:]]* vars \*/\) = 0
\+\+\+ exited with 0 \+\+\+
